### PR TITLE
Ability to set URL limit for specific domain

### PR DESCRIPTION
**Pull request by zaibacu**
Implementation of Feature request: https://github.com/crawler-commons/url-frontier/issues/90

### CODE REVIEW

File: service/src/main/java/crawlercommons/urlfrontier/service/QueueInterface.java [hunk 1]
```diff
@@ -23,4 +23,8 @@ public interface QueueInterface {
     public int getCountCompleted();
 
     public int countActive();
+
+    public void setCrawlLimit(int crawlLimit);
+
+    public Boolean IsLimitReached();
```
Comment (jnioche) [inline service/src/main/java/crawlercommons/urlfrontier/service/QueueInterface.java:29]: minor nitpick -> `isLimitReached` (lowercase initial i)

**Review by jnioche — CHANGES_REQUESTED** [review]
Minor change required but looks good otherwise. Thanks!

File: service/src/main/java/crawlercommons/urlfrontier/service/QueueInterface.java (hunk 1 above)
Comment (zaibacu) [reply service/src/main/java/crawlercommons/urlfrontier/service/QueueInterface.java:29]: Yeah, sorry, too much Golang lately :)

**Comment by jnioche** [comment]
thanks @zaibacu

**Pull request merged**